src/Workspaces/Core/Desktop/Workspace/Host/Mef/MefV1HostServices.cs

```diff
@@ -136,7 +136,9 @@ public override int GetHashCode()
 
         internal readonly struct TestAccessor
         {
+#pragma warning disable IDE0052 // Remove unread private members - hold onto the services
             private readonly MefV1HostServices _mefV1HostServices;
+#pragma warning restore IDE0052 // Remove unread private members
```

Comment:
why is this one needed?

Reply:
I saw these pattern in all TestAccessors, so I wasn't really sure if this was intentional or can be removed. @sharwell?